praas-app-ionic/src/components/Conduit/Create/index.tsx

```diff
@@ -4,6 +4,7 @@ import ConduitForm from '../Form';
 
 import { connect } from 'react-redux';
 import { addConduit } from '../../../store/conduit/create';
+import { useHistory } from 'react-router';
```

Comment:
being already a router component it already have access to history.

<img width="1065" alt="image" src="https://user-images.githubusercontent.com/62667177/88312884-029ae600-cd13-11ea-948f-9bf9620a8651.png">